app/controllers/api/songs_controller.rb

```diff
@@ -0,0 +1,102 @@
+class API::SongsController < API::APIController
+
+  before_action :require_edit_privileges, only: [:create, :update]
+  before_action :require_delete_privileges, only: [:destroy]
+
+  def show
+    song = Song.find_by(id: params[:id])
+    if song
+      case params[:format]
+      when 'numbers'
+        Formatter.format_song_nashville(song)
+      when 'no_chords'
```

Comment:
would it be more robust to parse `params[:format]` into some sort of enum and then switch on them?

Reply:
looks fine as is